manager/controllers/packages.go

```diff
@@ -34,12 +42,26 @@ func packagesQuery(acc int) *gorm.DB {
 	return database.Db.
 		Select(PackagesSelect).
 		Table("system_platform sp").
-		Joins("inner join rh_account ra on sp.rh_account_id = ra.id").
 		Joins("inner join system_package spkg on spkg.system_id = sp.id").
 		Joins("inner join package p on p.id = spkg.package_id").
 		Joins("inner join package_name pn on pn.id = p.name_id").
 		Where("spkg.rh_account_id = ?", acc).
-		Group("ra.id, pn.id, pn.name")
+		Group("spkg.rh_account_id, p.name_id, pn.name")
+}
+
+type packageSummary struct {
+	NameID  int
+	Summary string
+}
+
+func packageDescQuery(names []int) *gorm.DB {
+	return database.Db.
+		Select("distinct on(p.name_id) p.name_id, s.value").
```

Comment:
As it's not possible to compose compact query using gorm elements, we could use Raw element and composes standard  SQL script.